database_cleaner-core.gemspec

```diff
@@ -0,0 +1,30 @@
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "database_cleaner/version"
+
+Gem::Specification.new do |spec|
+  spec.name        = "database_cleaner-core"
+  spec.version     = DatabaseCleaner::VERSION
+  spec.authors     = ["Ben Mabey", "Ernesto Tagwerker"]
+  spec.email       = ["ernesto@ombulabs.com"]
+
+  spec.summary     = "Strategies for cleaning databases. Can be used to ensure a clean slate for testing."
+  spec.description = "Strategies for cleaning databases. Can be used to ensure a clean slate for testing."
+  spec.homepage    = "https://github.com/DatabaseCleaner/database_cleaner"
+  spec.license     = "MIT"
+
+  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.add_development_dependency "rake"
+  spec.add_development_dependency "bundler"
+  spec.add_development_dependency 'guard-rspec'
+  spec.add_development_dependency "listen"
+  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "cucumber"
+end
+
```

Comment:
Random empty line?

Reply:
Ah, I have a habit of ending all my files with carriage returns to make diffs cleaner when lines are added to the end. This file will probably never have that happen though, so I'll remove.

Reply:
This is two carriage returns, the posix standard has all lines end with `\n` where as this is `\n\n` or it wouldn't show up in github. Lines lacking a `\n` are equally invalid at the end of a file and are what cause the Github red symbol. A properly terminated file won't have any more diff than the new line you're adding.

Reply:
Wow, TIL! Awesome.